spec/controllers/rest/history_controller_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Rest::HistoryController do
+  let!(:repository) { create :repository_compound, :not_empty }
+  let!(:user) { repository.owner }
+  let!(:path) { repository.git.ls_files(revision).first }
+  let!(:revision) { repository.git.default_branch }
+
+  context 'successful (without revision)' do
+    describe 'action: index' do
+      before do
+        get :index, params: {organizational_unit_id: user.to_param,
+                              repository_id:
+                              repository.to_param.sub(%r{\A[^/]*/}, ''),
+                              path: path}
+      end
+      it { expect(response).to have_http_status(:ok) }
+      it { |example| expect([example, response]).to comply_with_rest_api }
+      it 'finds the repository' do
+        expect(response_data['repository']).not_to be(nil)
+      end
```

Comment:
Please add a test if the requested log contains all commits. You can get a list of all commits (Gitlab::Git::Commit objects) that change the file at `path` with
```ruby
repository.git.log(path: path)
```
If you are interested in the commit hashes instead, use
```ruby
repository.git.log(path: path).map(&:id)
```

Compare this to the commit ids that you receive from the REST API.